mlrun/config.py

```diff
@@ -75,6 +75,7 @@
     # sets the background color that is used in printed tables in jupyter
     "background_color": "#4EC64B",
     "artifact_path": "",  # default artifacts path/url
+    "model_endpoint_monitoring_container": "projects",
```

Comment:
I have a feeling this will grow, so add a section `model_endpoint_monitoring` under `httpdb` and inside add the `container`

Reply:
Resolved